newsfragments/3131.misc.rst

```diff
@@ -0,0 +1 @@
+Bootstrap uv using uv install script instead of pip
```

Comment:
```suggestion
Bootstrap uv using uv install script instead of pip, this works around an issue where pip refused to install uv on PyPy on windows
```